python/mrtrix3/commands/5ttgen/odf.py

```diff
@@ -0,0 +1,95 @@
+# HSVS SCRIPT
```

Comment:
```suggestion
```

(Hold off on merging: I'll make a bunch of suggestions on GitHub and they can all be merged in a single batch commit)